JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -479,34 +480,65 @@ private fun InterestsTabRow(
     updateSection: (Sections) -> Unit,
     tabContent: List<TabContent>
 ) {
-    // TODO: Make the tabs narrower and aligned to the left in large screens
-    TabRow(
-        selectedTabIndex = selectedTabIndex,
-        backgroundColor = MaterialTheme.colors.onPrimary,
-        contentColor = MaterialTheme.colors.primary
-    ) {
-        tabContent.forEachIndexed { index, tabContent ->
-            val colorText = if (selectedTabIndex == index) {
-                MaterialTheme.colors.primary
-            } else {
-                MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
+    BoxWithConstraints {
+        val windowSize = remember(maxWidth) { getWindowSize(maxWidth) }
```

Comment:
Just one comment here: since this `maxWidth` is in a nested component, the size classes will be slightly off from the size classes at the root level since the entire screen isn't available at this point.

For making "root-level" type decisions, having a composition local seems like it might be the way to go...

Reply:
Yeah, since the recommendation we want to give here is not 100% clear, I wanted to go for this implementation for now with the hope of revising all usages of `BoxWithConstraints` when we have a clearer guidance